CRM/Upgrade/Incremental/UserJob.php

```diff
@@ -0,0 +1 @@
+<?php
```

Comment:
@eileenmcnaughton is this meant to do anything? :-P

Reply:
@seamuslee001 argh - I fixed locally but only thought I pushed up